packages/commonwealth/client/scripts/controllers/server/sessions.ts

```diff
@@ -184,14 +198,15 @@ async function sign(
           actionMessage,
           actionMessageSignature,
         },
-        canvasHash: Buffer.from(sha256(encode(actionMessage))).toString('hex'),
+        canvasMsgId: actionMessageId,
       };
     }
   }
-  throw new Error(`No signer found for address ${address}`);
+  throw new Error(`No signer found for ${did}`);
 }
 
 // Public signer methods
+// TODO: rename all these to `createSignThreadAction()` or similar
```

Comment:
still a todo?

Reply:
Yeah, trying not to make any more changes at this stage.